x-pack/plugins/reporting/server/export_types/csv/create_job.ts

```diff
@@ -21,10 +21,12 @@ export const createJobFnFactory: CreateJobFnFactory<
     );
 
     const savedObjectsClient = context.core.savedObjects.client;
-    const indexPatternSavedObject = (await savedObjectsClient.get(
+    const {
```

Comment:
why are we not using plugins.data.indexpatterns to load ? (then we wouldn;t need to worry about this at all)